dev/com.ibm.ws.security.openidconnect.client/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -172,7 +172,7 @@ headerName=Name of the header containing the inbound token
 headerName.desc=The name of the header which carries the inbound token in the request.
 
 disableIssChecking=Disable issuer checking
-disableIssChecking.desc=Require the issuer claim to be absent when validating the OAuth token introspection response for inbound token propagation.
+disableIssChecking.desc=Require the issuer claim to be absent when validating a JWT access token for inbound propagation or when performing token introspection for inbound propagation.
```

Comment:
IBM style guidance is to avoid elliptical clauses like "when validating", 'when performing" as they can pose problems for translators. Usually the edit is to add a subject, like "when it validates" or "when you perform"

is it possible to add a subject for those verbs in this case-  maybe "you" ? or "the issuer" ?

Reply:
How do the updates sound?